crates/iceberg-catalog/src/service/storage/mod.rs

```diff
@@ -45,6 +47,29 @@ pub enum StorageType {
     Test,
 }
 
+// write_metadata expects Serialize + CommonMetadata, so for "test writes"
+// we use a TestMetadata struct with empty properties
+// TODO: define somewhere else?
```

Comment:
We could either give it a `#[cfg(test)]` attr or move it into the test module further down

Reply:
oh, I just realized we're using this for writing, I think location is fine then, if you're worried about it getting mixed up with other stuff you could move it into a `pub(crate) mod validation_data {}` or something

Reply:
Yes, I was kind of worried that it is "too visible"

Reply:
I ended up not moving it 😄